05/app.js

```diff
@@ -34,4 +34,37 @@ function insertUsers(usersList) {
 
         ulElement.appendChild(liElement);
     });
+    submit();
```

Comment:
Lepiej to wywołanie wrzucić do `init` - w tym miejscu inicjujemy wszystkie działania.
W tym miejscu raczej nikt by tego nie szukał ;)